bot/exts/utilities/bookmark.py

```diff
@@ -95,30 +89,40 @@ async def bookmark(
         *,
         title: str = "Bookmark"
     ) -> None:
-        """Send the author a link to `target_message` via DMs."""
-        if not target_message:
-            if not ctx.message.reference:
-                raise commands.UserInputError(
-                    "You must either provide a valid message to bookmark, or reply to one."
-                    "\n\nThe lookup strategy for a message is as follows (in order):"
-                    "\n1. Lookup by '{channel ID}-{message ID}' (retrieved by shift-clicking on 'Copy ID')"
-                    "\n2. Lookup by message ID (the message **must** be in the context channel)"
-                    "\n3. Lookup by message URL"
-                )
-            target_message = ctx.message.reference.resolved
+        """
+        Send the author a link to the specified message via DMs.
+
+        Members can either give a message as an argument, or reply to a message.
+
+        Bookmarks can subsequently be deleted by using the `bookmark delete` command.
```

Comment:
```suggestion
        Bookmarks can subsequently be deleted by using the `bookmark delete` command in DMs.
```

Reply:
https://github.com/python-discord/sir-lancebot/pull/1049/commits/0e4b03a9f2aa1d7a91dc3baa5a189ee4f9c380f5

Reply:
I could’ve sworn I wrote a comment on this, but it seems I didn’t. Anyways, this change is wrong because:
1. This command is decorated with a `guild_only`.
2. We don’t allow commands in DMs unless absolutely necessary.

And speaking of 1, that’s redundant since it’s the default for all commands.

Reply:
This is referring to the `bookmark delete` command, not the `bookmark` command. For the `bookmark delete` command the defaults were overwritten to allow it in (and only in) DMs.

Reply:
In that case we should probably reconsider. It's policy not to do this, and with decent reason. Core-devs please refer to the [policy doc](https://www.notion.so/pythondiscord/Internal-Policies-86f26339ba86483a90b1b6dd1d157886).

Reply:
Having to jump between DMs and the server to delete a bookmark is not great UX, so in this case I think it makes sense to allow it in DMs. It's a very simple command (implementation) so I don't think there should be any issues caused by this.

Reply:
I don't think this is better UX, considering all that comes with this (Chris left a comment about it below).

Reply:
When you book mark a command, the link to that message is sent to you in DM. The best UX is to allow users to delete the message directly from the same place.

The UX issue described below is related to the help command, rather than this command directly.

We have two options to fix that, We could allow the help command to be ran in DMs, which leads to the screenshot I posted about below, or we allow the bookmark delete command to also be ran in server, so at least users can get the help info about it.

Regardless of options, i believe this is a very valid use case for a command that can be ran in DMs.